classes/01class/exercises/c01-aws01/fabioacp/ANSWER.md

```diff
@@ -0,0 +1,54 @@
+# C01-AWS01
+
+## Commands Execution Output
+
+- Commands to create the first EC2 instance and any additional resource required:
+```bash
+# Create a KeyPair
+aws ec2 create-key-pair --key-name MyKeyPair --query 'KeyMaterial' --output text > MyKeyPair.pem
+
+# Create a EC2 instance with a public IP
+aws ec2 run-instances \
+    --image-id ami-088ff0e3bde7b3fdf \
+    --instance-type t2.micro \
+    --count 1 \
+    --subnet-id subnet-a50917ec \
+    --key-name MyKeyPair \
+    --security-group-ids sg-0983070dc820d22e6 \
+    --associate-public-ip-address
+```
+
+- Commands to connect to the first EC2 instance:
+```bash
+#Set Permission
+CHMOD 400 MyKeyPair.pem
```

Comment:
can you check if this command worked?

Reply:
Yes, we can use the command:
❯ ls -l MyKeyPair.pem 
-r--------  1 fabiopereira  staff  1671 14 Jun 20:59 MyKeyPair.pem.

or If you talking about the command "CHMOD", it may be case-sensitive on Ubuntu but it worked for me.

Reply:
ouch.my bad.. just confirmed that MacOS accepts as uppercase as well.. :facepalm: